Code-the-Dream-School/dd-prac-team1-back
# Feat/save into db pra 82

## One Line Summary
Update openai schema, renaming certain fields, to align with our recipe schema for saving recipe data to the database.

## Requirements
Rename specific fields on the front end to ensure proper functionality.

_What the Code Should Accomplish_
Create an endpoint, api/v1/ai-recipe/add, that allows users to save openai data to the database.

## Notes

_Any additional information_

## Test Steps

_Instructions for testing the changes_

/- Guide:
/- After making the PR, perform npm install.

Add a recipe by setting the request method as POST and utilizing the URL of your backend API's recipe addition endpoint (e.g., http://localhost:3000/api/v1/ai-recipe/add).
/- For requests, include the necessary headers (e.g., Content-Type: application/json) to indicate that JSON data is being sent in the request body.
In the request body, provide sample data for the required fields to test the endpoints. Ensure that the data is valid and adheres to any validation rules in place.
/- Click "Send Request" for each request to send the HTTP request to the backend server.
/- Check the response received for any errors or status codes.
/- For successful additions, the response should have a successful status code (e.g., 201 Created) and a JSON object containing recipe data.

## Checklist

- The Linear ID is mentioned in the PR title.
- My code adheres to best practices.
- Documentation is provided where necessary.
